### ISSUE TITLE
Suggestion: organisation of resources links

### ISSUE THREAD

**Issue by JamesChristie-SustainableUX** [w3c/sustyweb#26]
The current document has lots of resources / citation links, which is great. Some of the lists are unwieldy or have other issues.

Issue 0: no visited-link CSS color. 
Many links repeat across sections, but because a:visited isn't implemented in the CSS, the visitor can't tell they've already opened that tab. 
BTW if it isn't already, this could be a tip for 2.6 or 2.8 - preventing unessessary / unwanted site visits by hinting the user already went there. 

Issue 1: mystery links: 
Example on 2.19:  [GR491] 2-4012, 3-4034, and 9-5064
Ideally spell out the content, e.g. [Do all images have a text alternative?](https://gr491.isit-europe.org/en/crit.php?id=9-5064-frontend-accessibility-assistants-must-be-able-to-use) - Handbook of sustainable design

Issue 2: surprise! PDF:
Especially on mobile, it is unwelcome to open PDFs unexpectedly. Perhaps we could use a format like "Link name" Publication (PDF).
Likewise, if the link lead to a video. 

Issue 3: long lists of resources with mixed topics
Random example: 2.19 has 20 resources, listed alphabetically. 
These could be broken up by category:
- General guidance
- Downloads
- Fonts
- Text alternatives
- Video captions and transcripts

Issue 4: Lots of resources, variable quality.
In some cases it looks like there is a lot of overlap in resources. Perhaps there could be a culling.
I think users might appreciate seeing the source, too - there's many differences between one persons medium post vs an authoritative guideline. Both are valid, but if the visitor is going to open 4 links out of 20 they might want to select both formal and informal resources. 
It might also be helpful to pull out 1 or 2 resources that either give broad guidance on all the relevant issues or are especially high quality.

**Comment by AlexDawsonUK** [comment]
Thanks for adding each of these link related issues, I'll address them in turn:

0. The W3C tool we use to format the specification (ReSpec) by default is quite ridged in terms of formatting, however it does allow for some customization of the CSS so I'll definitely look into this to see if it can be applied. I also agree there is potential for it being mentioned within one of the guidelines so I'll make a note of this.
1. I agree mystery links are problematic, however the only times we have used them (shown in the example you used) is within a recognized best-practice format or standard. We'll discuss regarding formatting such content better though.
2. Agree, this will be resolved in the next draft.
3. Alphabetizing was done to avoid making a large document a LOT larger, having a more structured formatting of the content could be done if the need was there though.
4. I feel less certain about this point. We don't endorse third party links (as we don't control the content), and while it's true that some links may be more formal than others, they were included as they offered something of value. Granted information overload can be an issue but if the information is valid, we don't wish to discriminate upon the source.

**Comment by AlexDawsonUK** [comment]
**Update:** ReSpec does allow for some reformatting which can potentially offer a hacky solution for visited links but it's far from ideal, and more to the point, there are [security implications](https://developer.mozilla.org/en-US/docs/Web/CSS/Privacy_and_the_:visited_selector) involved in changing visited state (even changing color can be detected using tooling which could then identify through a script an individuals browser history). As such we shall not include this within the next draft as it poses a browser security concern (some potential issues still remain).

**Comment by AlexDawsonUK** [comment]
Each of the issues documented above have been addressed:

0. Realistically this needs to be pulled up as an issue with the [ReSpec](https://github.com/w3c/respec/issues) team as the default styled are theirs, however as a temporary patch I've provided a color workaround which should avoid additional issues you're encountering.
1. Each link from authority (such as a specification) now has a numbered value plus a short text description.
2. All PDF's and videos are identified as such besides the link.
3. This is still under discussion and depending on how links are formatted and managed later on, it may affect how we choose to progress with this issue so for the moment it will be left as unresolved.
4. As noted in my previous comment, we are not the arbiters of authority nor do we wish to discredit accurate information wherever it may originate (even though we don't directly endorse linked-to content), so this will not be progressed.

**Summary:** Issue 0, 1, & 2 = ✅. Issue 3 & 4 = ❎. Updates will appear when the next draft is published.
